.markdownlint.yml

```diff
@@ -11,6 +11,7 @@
 "MD026": false # Trailing punctuation in header
 "MD036": false # Emphasis instead of header
 "MD046": false # Code block style. Doesn't like MkDocs admonitions or content tabs
+"MD051": false # Link fragments should be valid. Reports false negatives when using custom anchor for subsections.
```

Comment:
```suggestion
```

We want to avoid this, as it should prevent us getting broken internal links. Just need to work out what the issue is.